src/main/resources/org/jenkinsci/plugins/docker/workflow/Docker.groovy

```diff
@@ -103,13 +104,13 @@ class Docker implements Serializable {
         public <V> V inside(String args = '', Closure<V> body) {
             docker.node {
                 try {
-                    docker.script.sh "docker inspect -f . ${id}"
+                    docker.script.sh "docker inspect -f . ${imageName()}"
```

Comment:
This is wrong I think. The point of the preinspection is to determine if we can run from the local cache. (This is important for this plugin because otherwise `docker-run` might block on a network operation, which would interfere with durability and progress guarantees for Pipeline.)